automation/rancher-partner-charts/app-readme.md

```diff
@@ -0,0 +1,35 @@
+## Introduction
```

Comment:
@qbert2k it seems to not belong to this PR. Changes in `automation/rancher-partner-charts`.